src/caret_analyze/infra/lttng/architecture_reader_lttng.py

```diff
@@ -17,6 +17,7 @@
 from collections.abc import Sequence
 
 
+from .lttng import Lttng
 from .lttng import LttngEventFilter
```

Comment:
The two import statements can be merged.
```suggestion
from .lttng import Lttng, LttngEventFilter
```

Reply:
I fixed it